README.md

```diff
@@ -68,6 +68,7 @@ class ExampleServiceClient
       error_threshold:  50,
 
       # seconds before the circuit times out
+      # if set to nil no timeout is used
```

Comment:
It makes sense to keep this to what ruby's timeout method accepts, which is [nil or 0](http://ruby-doc.org/stdlib-2.4.0/libdoc/timeout/rdoc/Timeout.html#method-c-timeout) for no timeout. I can easily add a check for zero so we skip the timeout codepath in circuitbox.

We could technically ignore most of the work i've done in this PR and just document that the values are passed directly to timeout that way ```timeout_seconds``` can be set to nil or 0 to run with no timeout.

Ideally I would rather just remove all of the timeout use in circuitbox and have users of the gem handle timeouts themselves but there's been quite a bit of pushback when I tried to do that, #65